Tests/Unit/DependencyInjection/CmfSeoExtensionTest.php

```diff
@@ -121,11 +121,13 @@ public function testAlternateLocaleWithPhpcr()
             'Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\AlternateLocaleProvider'
         );
 
-        $this->assertContainerBuilderHasServiceDefinitionWithMethodCall(
-            'cmf_seo.event_listener.seo_content',
-            'setAlternateLocaleProvider',
-            array($this->container->getDefinition('cmf_seo.alternate_locale.provider_phpcr'))
-        );
+        if ($this->container->has('cmf_seo.event_listener.seo_content')) {
```

Comment:
Do then you should use the config section above of the test and check for the existence by on or  many asserts. Cause i think your/my event listener will exist only, when there is a line in the configuration, right?